src/hub/action_request.ts

```diff
@@ -164,6 +164,7 @@ export class ActionRequest {
 
     const streamPromise = new Promise<void>((resolve, reject) => {
       if (url) {
+        process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0"
```

Comment:
should just set this in your local environment for testing if you need it so it doesn't sneak in like this

Reply:
ah yes thanks for catching this